spras/containers.py

```diff
@@ -109,24 +110,31 @@ def prepare_dsub_cmd(flags: dict):
 
     # iteratively add flags to the command
     for flag in ordered_flags.keys():
-        if isinstance(ordered_flags.get(flag), list):
-            for f in ordered_flags.get(flag):
+        value = ordered_flags.get(flag)
```

Comment:
[Note: I moved this to a general `value` to satisfy pyright's type checker, as it can not determine if the `.get` function is pure over the currently unmutated `ordered_flags`.]